src/settings-ui/Settings.UI/SettingsXAML/App.xaml.cs

```diff
@@ -208,17 +209,17 @@ protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs ar
             }
             else
             {
+#if DEBUG
                 // For debugging purposes
                 // Window is also needed to show MessageDialog
                 settingsWindow = new MainWindow(isDark);
                 settingsWindow.ExtendsContentIntoTitleBar = true;
                 settingsWindow.Activate();
                 settingsWindow.NavigateToSection(StartupPage);
-
-#if !DEBUG
```

Comment:
This line and the removed line below changes debug/no debug behavior and moves thing into debug case. Mistake?

Reply:
no. We still want to run it in Debug mode, for development purposes. In Release mode we start it properly via runner. Before, only difference in behavior was that in Debug mode it keeps running, and in Release it shows the error dialog box and exits.